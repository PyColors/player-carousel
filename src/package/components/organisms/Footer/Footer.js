import React from "react";

import Row from 'muicss/lib/react/row';
import Col from 'muicss/lib/react/col';

import logoBottom from '../../../../../public/bt_logo.png';

const Footer = () => (
    <footer className='footer'>
            <Row>
                <Col md="6"><img className='footer__logo' src={logoBottom} alt='BT'/>
                    <p className='footer__title'>BT TV</p>
                    <span className='footer__copyright'>@copyright 2018</span></Col>
                <Col md="6" mui--text-right>
                    <div className="mui--text-right">
                        <img className='footer__travis--logo' src="https://travis-ci.org/PyColors/player-carousel.svg?branch=master"
                             alt="Build Status"/>
                        <a className="footer__travis--text" href="https://github.com/PyColors/player-carousel"
                           target="_blank">Project on
                            GitHub</a>
                    </div>

                </Col>
            </Row>
    </footer>
);

export default Footer;
