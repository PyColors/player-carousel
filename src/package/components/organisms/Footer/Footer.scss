.inline-middle {
  display: inline-block;
  vertical-align: middle;
}

.footer {
  margin-bottom: 20px;

  &__logo {
    width: 3%;
    @extend .inline-middle;
  }

  &__title {
    @extend .inline-middle;
    margin: 0 0 0 4px;
    font-family: "BT Regular", serif;
    font-size: 12px;
    color: #fff;
  }

  &__copyright {
    @extend .inline-middle;
    margin: 0 0 0 20px;
    font-family: "BT Light", serif;
    font-size: 11px;
    color: #5D5A62;
  }

  &__travis--logo {
    @extend .inline-middle;
  }

  &__travis--text {
    @extend .inline-middle;
    margin: 0 0 0 10px;
    font-family: "BT Light", serif;
    font-size: 11px;
    color: #5D5A62;
  }

}
