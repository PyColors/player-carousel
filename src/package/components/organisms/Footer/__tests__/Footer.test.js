import React from "react";
import Footer from "../Footer";

it("Should renders Footer page without exploding", () => {
    const wrapper = shallow(<Footer>Hello Jest!</Footer>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveLength(1)
});
