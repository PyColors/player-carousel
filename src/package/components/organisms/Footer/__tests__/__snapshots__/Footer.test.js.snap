// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders Footer page without exploding 1`] = `
<footer
  className="footer"
>
  <Row
    className=""
  >
    <Col
      className=""
      lg={null}
      lg-offset={null}
      md="6"
      md-offset={null}
      sm={null}
      sm-offset={null}
      xl={null}
      xl-offset={null}
      xs={null}
      xs-offset={null}
    >
      <img
        alt="BT"
        className="footer__logo"
        src="test-file-stub"
      />
      <p
        className="footer__title"
      >
        BT TV
      </p>
      <span
        className="footer__copyright"
      >
        @copyright 2018
      </span>
    </Col>
    <Col
      className=""
      lg={null}
      lg-offset={null}
      md="6"
      md-offset={null}
      mui--text-right={true}
      sm={null}
      sm-offset={null}
      xl={null}
      xl-offset={null}
      xs={null}
      xs-offset={null}
    >
      <div
        className="mui--text-right"
      >
        <img
          alt="Build Status"
          className="footer__travis--logo"
          src="https://travis-ci.org/PyColors/player-carousel.svg?branch=master"
        />
        <a
          className="footer__travis--text"
          href="https://github.com/PyColors/player-carousel"
          target="_blank"
        >
          Project on GitHub
        </a>
      </div>
    </Col>
  </Row>
</footer>
`;
